packages/syft/src/syft/client/client.py

```diff
@@ -527,6 +528,29 @@ def create_project(
         project = project_create.start()
         return project
 
+    def sync_code_from_request(self, request):
+        code = deepcopy(request.code)
+        code.worker_pool_id = None
```

Comment:
`code.worker_pool_id` has been replaced with `code.worker_pool_name`.
Also, any reason we're setting worker_pool_id to None ? Don't we want the code to be executed on custom workers ?